Assets/Scripts/Navigation/NavigationController.cs

```diff
@@ -71,18 +68,24 @@ void LeftPanelButtonClicked()
         }
     }
 
-    // Button Listener: Filter
-    void AddFilterButtonClicked()
+    // The exit button clicked: Delete all QM objects under "State"
+    private void ExitButtonClicked()
     {
-        FilterWindow.SetActive(true);
+        GameObject stateGameObject = GameObject.Find("State");
```

Comment:
instead of destroying objects, I'd quit the application using `Application.Quit` (see [docs](https://docs.unity3d.com/ScriptReference/Application.Quit.html))